app/scripts/modules/core/src/pipeline/config/stages/stage.html

```diff
@@ -50,7 +50,7 @@ <h4 ng-bind="stage.name || '[new stage]'"></h4>
         <span class="glyphicon glyphicon-trash" uib-tooltip="Remove stage"></span> Remove stage
       </button>
       <button type="button" class="btn btn-sm btn-default" ng-click="stageConfigCtrl.editStageJson(stage)">
-        <i class="fa fa-cog"></i> Edit stage as JSON
+        <loading-spinner size="'nano'"></loading-spinner> Edit stage as JSON
```

Comment:
Not this one - it's an edit button, not a loading/progress indicator

Reply:
Good catch :) thank you